content/docs/cucumber/api.md

```diff
@@ -57,9 +73,12 @@ Given("the following animals:") { animals: java.util.List[String] =>
 ```
 {{% /text %}}
 
-In this case, the `DataTable` is automatically flattened to a {{% text "java,kotlin" %}}`List<String>`{{% /text %}}{{% text "scala" %}}`java.util.List[String]`{{% /text %}}
+In this case, the `DataTable` is automatically flattened to a {{% text "java,kotlin" %}}`List<String>`{{% /text %}}{{% text "scala" %}}`java.util.List[String]`{{% /text %}}{{% text "ruby,javascript" %}}list of strings{{% /text %}}
```

Comment:
Minor: If one were a stickler for details, one _could_ call the Ruby & JS list an _array._

Reply:
Updated, thnx